recipes/example/meta.yaml

```diff
@@ -9,12 +10,13 @@
 #  `conda install openssl -c conda-forge``
 
 package:
-  name: simplejson
+  name: {{ name|lower }}
```

Comment:
Hmm...I thought about this and recently went the wrong way. In particular, I thought `lazy-object-proxy` should have been (or was intended to be) `lazy_object_proxy` and found out that it is not the case. There may be other examples.